scripts/setup-adapters.sh

```diff
@@ -46,6 +46,18 @@ function install_libhdfs3 {
   cmake_install
 }
 
+function install_protobuf {
+  yum -y install gcc-c++
+  yum -y remove protobuf*
```

Comment:
Is this required? Users usually run these scripts without reading. It might be undesired for Velox scripts to remove exisiting libraries.

Reply:
> Is this required? Users usually run these scripts without reading. It might be undesired for Velox scripts to remove exisiting libraries.

It's not required, I remove this.